tests/D2L.CodeStyle.TestAnalyzers.Tests/IgnoreAttribute/IgnoreAnalyzerTests.cs

```diff
@@ -37,6 +37,24 @@ class Test {
 		}
 
 		[Test]
+		public void DocumentWithoutTestIgnore_NoDiag() {
+			const string test = @"
+	using System;
+
+	namespace test {
+		[ReflectionSerializer]

```

Comment:
I update the analyzer as your comments. Leaving this one as a test case that the ignore is not on a test method still makes sense?